### ISSUE TITLE
Setting escape-time to 0 makes bindings using alt/meta not always work

### ISSUE THREAD

**Issue by anowlcalledjosh** [tmux-plugins/tmux-sensible#41]
I have the following in my .tmux.conf to switch windows with alt-h and alt-l:

```
bind -n M-h previous-window                                                      
bind -n M-l next-window
```

and sometimes with `escape-time` set to 0 by tmux-sensible, then `^[h` or `^[l` will be echoed in the terminal rather than switching windows. Setting `escape-time` to 10 seems to fix the problem without adding any noticeable delays when pressing esc elsewhere (for example in Vim) – could this be made the default in tmux-sensible?

**Comment by linden-slam** [comment]
Supporting this issue also due to [this](https://superuser.com/a/1809494/224038) StackOverflow answer.

**TL;DR**: `escape-time` should never be set to `0`, but rather something small like a value between `5` and `100`. Otherwise it can cause weird bugs where escape sequences fragmented over multiple packets are not recognized properly.
